lib/standard/stream.nit

```diff
@@ -392,6 +392,34 @@ abstract class BufferedReader
 		return c.ascii
 	end
 
+	# Peeks up to `n` bytes in the buffer, returns an empty string on EOF
+	#
+	# The operation does not consume the buffer
+	#
+	# ~~~nitish
+	# 	var x = new FileReader("File.txt")
+	#	assert x.peek(5) == x.read(5)
+	# ~~~
```

Comment:
> I'm afraid I cannot do that Dave.

Since we're in a Nitunit, wouldn't a unit test opening a file be too fragile ? I plan on changing it to a executable test once the StringBuffer becomes a BufferedReader (which should be possible once the buffer of the BufferedReader is a simple NativeString, since it is not modified internally it can be a simple proxy to the items of a String)

Furthermore, there is already an executable test to ensure that peek functions adequately until I can do a nitunit for that.

But yup, the `new FileReader.open` will need to be rerolled right now.